include/magma/vk/buffers/Buffer.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file Buffer.h
+ * @brief Class containig all information about buffer
+ * @version 0.1
+ * @date 2021-02-19
+ */
+#pragma once
+#include <string>
+#include <vulkan/vulkan.h>
+#include <vulkan/vulkan_core.h>
+
+struct BufferInfo {
+    VkDevice                device;
+    VkBufferCreateInfo      bufferInfo;
+    VkMemoryPropertyFlags   memoryProperty;
+    std::string             name;
+};
+
+class Buffer {
+friend class BufferManager;
+
+public:
+    Buffer() = default;
+    const VkBuffer          &getBuf()   const { return _buffer;         }
+    const VkDeviceMemory    &getMem()   const { return _bufferMemory;   }
+    BufferInfo*             getInfo()   const { return _info;           }
```

Comment:
`_info` is always non-null, so i think it will be better to return `const` reference

Reply:
Anyway `const` qualifier should be added to return type since we don't want to change `BufferInfo` manually